### PR TITLE
Update desktop-changelog.rst

### PR DESCRIPTION
**Pull request by amyblais**
Adding a note about potential desktop app dot release that the SET team is looking into.

### CODE REVIEW

File: source/help/apps/desktop-changelog.rst [hunk 1]
```diff
@@ -6,6 +6,9 @@ Release v4.6
 
 **Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/tag/v4.6.1>`_
 
+
+- **v4.6.2, release day TBD**
+ - Investigating fixing an issue where logging in to ``gitlab.com`` does not work on the desktop apps. `MM-31626 <https://mattermost.atlassian.net/browse/MM-31626>`_
```
Comment (cwarnermm) [inline source/help/apps/desktop-changelog.rst:11]:
```suggestion
 - Investigating fixing an issue where logging in to ``gitlab.com`` does not work on the Desktop App. `MM-31626 <https://mattermost.atlassian.net/browse/MM-31626>`_
```

Comment (cwarnermm) [reply source/help/apps/desktop-changelog.rst:11]: Is this issue present on a specific version(s) of the Desktop App (such as 4.6.1)? If so, consider adding that detail to the bullet point as well. If adding version details, you can omit "the" -- example: "does not work on Desktop App [version]."

**Review by cwarnermm — COMMENTED** [review]
Minor edits suggested inline.

File: source/help/apps/desktop-changelog.rst (hunk 1 above)
Comment (amyblais) [reply source/help/apps/desktop-changelog.rst]: This issue affects all versions.

**Review by cwarnermm — APPROVED** [review]
Looks good to me!

**Pull request merged**